inference-engine/samples/benchmark_app/main.cpp

```diff
@@ -283,6 +283,21 @@ int main(int argc, char *argv[]) {
 
                 if (isFlagSetInCommandLine("nthreads"))
                     device_config[GNA_CONFIG_KEY(LIB_N_THREADS)] = std::to_string(FLAGS_nthreads);
+            } else {
```

Comment:
As far as I understood this code is works for device with names different from `CPU`. I mean for some custom devices which supports CPU settings. Is it correct?

Reply:
Yes, it is correct. Configuration options from the PR are used by IE Streams CPU Executor which can be used by any plugin not only CPU one. So if the device plugin support such options it means that it configure own instance of Streams Executor internally.